src/swap/SwapExecuteScreen.tsx

```diff
@@ -46,7 +34,7 @@ export function SwapExecuteScreen() {
           <View style={styles.actionBar}>
             <Button
               text={t('SwapExecuteScreen.swapActionBar.tryAgain')}
-              onPress={navigateToReviewScreen}
+              onPress={navigateBack}
```

Comment:
sorry, this is an existing issue but i didn't notice until now...do you think we should add analytics for these buttons? :/ i'm not sure, just noting it because i noticed it

Reply:
I agree, let's at least record if any of those buttons are tapped.
Added analytics events for "Swap again" and "Try again" taps.